storybook/src/components/Radio/Radio.stories.tsx

```diff
@@ -145,3 +149,9 @@ export const InAFieldSetWithValidation: Story = {
     </FieldSet>
   ),
 }
+
+export const CustomIcons: Story = {
+  args: {
+    icon: <FavouriteIcon />,
```

Comment:
Too bad it doesn't have children to show a selected state. It would be cool if you are able to put an SVG in here instead of an icon component.

Reply:
Yes, please give me a heart-shaped dot when I click it! 😍

Reply:
I updated this. Maybe not quite what you envisioned, but close